mopidy_musicbox_webclient/static/mb.manifest

```diff
@@ -1,6 +1,6 @@
 CACHE MANIFEST
 
-# 2016-01-31:v3
+# 2016-02-01:v1
 
 NETWORK:
 *
```

Comment:
Despite the noise it generates I'm not against trying to keep doing this, I can just see myself forgetting.  I guess all we *really* need to ensure is that the releases that we package / tag get the treatment.

Or maybe Travis could get off his arse and check the manifest as part of his build :)